website/content/api-docs/secret/pki.mdx

```diff
@@ -106,6 +113,321 @@ to mix different types of CAs (roots and intermediates).
+
+#### ACME External Account Bindings
+
+ACME External Account Binding (EAB) Policy can enforce that clients need to
+have a valid external account binding. Before registering a new account,
+an authenticated Vault client will need to [fetch a new EAB
+token](#get-acme-eab-binding-token). This returns two values: a key identifier
+and an HMAC key used by the ACME client to authenticate with EAB. For example:
+
+```
+$ vault write -f /pki/acme/new-eab
+$ certbot certonly --server https://localhost:8200/v1/pki/acme/directory \
+                   --eab-kid <id> --eab-hmac-key <hmac-key>
+```
+
+With or without EAB, requests from the ACME client are not authenticated using
+traditional Vault authentication, but are instead authenticated through the
+ACME protocol. With EAB however, a Vault authenticated client will have to
+fetch an EAB token and pass it to the ACME client for use on the initial
+registration: this binds the ACME client's registration to an authenticated
+Vault endpoint, but not further to the client's entity or other information.
+
+~> Note: Enabling EAB is strongly recommended for public-facing Vault
+   deployments. Use of the `VAULT_DISABLE_PUBLIC_ACME` environment variable
+   can be used to enforce all ACME instances have EAB enabled.
+
+#### ACME Accounts
+
+ACME Accounts are created specific to a particular directory and are not
+portable across Performance Secondary clusters.
+
+#### ACME Required Headers
+
+ACME requires the following response headers (`allowed_response_headers`)
+to be specified by [mount tuning](/vault/api-docs/system/mounts#tune-mount-configuration):
+
+ - `Replay-Nonce`
+ - `Link`
+ - `Location`
+
+On an existing mount, these can be specified by running the following command:
+
+```
+$ vault secrets tune -allowed-response-headers=Location -allowed-response-headers=Replay-Nonce \
+                     -allowed-response-headers=Link \
+                     pki/
+```
+
+### Get ACME EAB Binding Token
+
+This endpoint returns a new ACME binding token. The `id` response field can
+be used as the key identifier and the `key` response field be used as the
+EAB HMAC key in the ACME Client.
+
+Each call to this endpoint will generate and return a new EAB binding token.
+
+| Method | Path                |
+| :----- | :------------------ |
+| `POST` | `/pki/acme/new-eab` |
+
+#### Parameters
+
+No parameters.
+
+#### Sample Request
+
+```
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request POST \
+    http://127.0.0.1:8200/v1/pki/acme/new-eab
+```
+
+#### Sample Response
+
+```
+{
+  "data": {
+    "created_on": "2023-05-24T14:33:00-04:00",
+    "id": "bc8088d9-3816-5177-ae8e-d8393265f7dd",
+    "key_bits": "256",
+    "key_type": "hs",
+    "key": "MHcCAQE... additional data elided ...",
+  }
+}
+```
+
+### List Unused ACME EAB Binding Tokens
+
+This endpoint returns a list of all unused ACME binding tokens; once used,
```

Comment:
This is authed, right? (maybe say, "this authenticated endpoint")

Reply:
We usually only list unauthenticated endpoints; its assumed endpoints are authed by default. I've added a missing note to the Directories that it is unauthed from a Vault auth perspective though. :-) thanks!